src/defaults.mligo

```diff
@@ -13,18 +13,23 @@ let validate_default_config (data : initial_config_data) : unit =
 
 let default_config (data : initial_config_data) : config =
   let u : unit = validate_default_config(data) in {
-    proposal_check = (fun (params, extras : propose_params * contract_extra) -> true);
-    rejected_proposal_return_value = (fun (proposal, extras : proposal * contract_extra) -> 0n);
-    decision_lambda = (fun (proposal, extras : proposal * contract_extra) -> (([] : (operation list)), extras));
-    quorum_threshold = data.quorum_threshold;
-    fixed_proposal_fee_in_token = data.fixed_proposal_fee_in_token;
-    voting_period = data.voting_period;
-    max_proposals = 500n;
-    max_votes = 1000n;
-    proposal_flush_time = data.proposal_flush_time;
-    proposal_expired_time = data.proposal_expired_time;
-    custom_entrypoints = (Big_map.empty : custom_entrypoints);
-  }
+  proposal_check = (fun (params, extras : propose_params * contract_extra) -> true);
+  rejected_proposal_return_value = (fun (proposal, extras : proposal * contract_extra) -> 0n);
+  decision_lambda = (fun (proposal, extras : proposal * contract_extra) -> (([] : (operation list)), extras));
+  fixed_proposal_fee_in_token = data.fixed_proposal_fee_in_token;
+  voting_period = data.voting_period;
+  max_proposals = 500n;
+  max_votes = 1000n;
+  max_quorum_threshold = data.max_quorum;
+  min_quorum_threshold = data.min_quorum;
+  max_quorum_change = data.max_quorum_change;
+  quorum_change = data.quorum_change;
+  governance_total_supply = data.governance_total_supply;
+  proposal_flush_time = data.proposal_flush_time;
+  proposal_expired_time = data.proposal_expired_time;
+
+  custom_entrypoints = (Big_map.empty : custom_entrypoints);
+}
```

Comment:
Nitpick, probably this is due to the rebase, but this seemed better with the previous indentation.

Reply:
Fixed.